intel/docs/SGX-gateway.md

```diff
@@ -55,7 +56,7 @@ $ make docker
 
 3. Install Istio
 
-> NOTE: for the below command you need to use the `istioctl` for the `docker.io/intel/istioctl:1.17.1-intel.2` since only that contains Istio manifest enhancements for SGX mTLS.
+> NOTE: for the below command you need to use the `istioctl` for the `docker.io/intel/istioctl:1.19.0` since only that contains Istio manifest enhancements for SGX mTLS.

```

Comment:
```suggestion
> NOTE: for the below command you need to use the `istioctl` for the `docker.io/intel/istioctl:1.19.0-intel.0` since only that contains Istio manifest enhancements for SGX mTLS.

```
For istio images, we still use names like 1.19.0-intel.0